File: Selenium/Activity1.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService

with webdriver.Firefox() as driver:

 driver.get("https://training-support.net/")

print("Page title is: ", driver.title)

driver.find_element(By.LINK_TEXT, "About Us").click()

print("New page title is: ", driver.title)

File: TestNG/Activities/fst-m1-testng/src/test/java/Activities/Activity1.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;


public class Activity1 {
	WebDriver driver;
	
	@BeforeClass
	public void setUp() {
		
		driver=new FirefoxDriver();
		driver.get("https://training-support.net");
	}
	@Test
	public void Title() {
		Assert.assertEquals(driver.getTitle(),"Training Support");
        driver.findElement(By.linkText("About Us")).click();

		
	}
	@Test(priority = 2)
    public void aboutPageTest() {
        Assert.assertEquals(driver.getTitle(), "About Training Support");
    }
    
    @AfterClass
    public void tearDown() {
        driver.quit();
	

}
}
